src/lib/bits/pin-input/components/pin-input-input.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+	import { melt } from "@melt-ui/svelte";
+	import { getCtx, getAttrs } from "../ctx.js";
+	import type { InputEvents, InputProps } from "../types.js";
+	import { createDispatcher } from "$lib/internal/events.js";
+
+	type $$Props = InputProps;
+	type $$Events = InputEvents;
+
+	export let asChild: $$Props["asChild"] = false;
+
+	const {
+		elements: { input }
+	} = getCtx();
+
+	const dispatch = createDispatcher();
+	const attrs = getAttrs("input");
+
+	$: builder = $input();
```

Comment:
I spent so much time trying to figure out what was wrong here when I used `$input` instead of `$input()` 🤦